### DISCUSSION TITLE
I miss Fennec's UI

### DISCUSSION THREAD

**Discussion by cassepipe** [mozilla-mobile/fenix#28431] [General]
I miss Fennec's UI so much. It was great.
Is there still a way to run it ? Who designed it ?

**Comment by anewuser** [comment]
The Firefox app has been rewritten from scratch. Using the old app today would be a security risk.

Please use [Mozilla Connect](https://connect.mozilla.org/) to vote for the features that you miss the most. Maybe you can start by supporting this idea: [Bookmarks on the Home Page](https://connect.mozilla.org/t5/ideas/bookmarks-on-the-home-page-android/idi-p/5517).
